src/ape/api/accounts.py

```diff
@@ -147,6 +147,8 @@ def deploy(self, contract: "ContractContainer", *args, **kwargs) -> "ContractIns
         address = click.style(receipt.contract_address, bold=True)
         logger.success(f"Contract '{contract.contract_type.contractName}' deployed to: {address}")
 
+        from ape.contracts import ContractInstance
```

Comment:
Can this be put at the top? or does that create a cycle?

Reply:
That creates the cycle... I tried looking into other ways to fix, but they just need each other because a contract _is_ an address type and the `deploy` method returns stuff related to contracts. It is very cyclic